setup.py

```diff
@@ -161,6 +164,7 @@ def write_version(filename=os.path.join(*['airflow',
 databricks = ['requests>=2.5.1, <3']
 datadog = ['datadog>=0.14.0']
 doc = [
+    'mock',
```

Comment:
?? This is needed for docs? I would have though mock is just used in tests/, and i’m Surprised it’s needed here?

Reply:
I did some digging and looks like we use it here:
https://github.com/apache/incubator-airflow/blob/acca61c602e341da06ebee2eca3a26f4e7400238/docs/conf.py#L16

This is used for mocking import of various modules:
https://github.com/apache/incubator-airflow/blob/master/docs/conf.py#L18-L31


Reference: http://blog.rtwilson.com/how-to-make-your-sphinx-documentation-compile-with-readthedocs-when-youre-using-numpy-and-scipy/

Reply:
@Fokko What Kaxil found is consistent with what I saw as well.  There's a bit more info on it in (1) in my comment here https://github.com/apache/incubator-airflow/pull/3703#issuecomment-410836135.